readme.md

```diff
@@ -325,9 +325,10 @@ A `streamName` can be passed to pipe `"stderr"`, `"all"` (both `stdout` and `std
 
 Returns `execaChildProcess`, which allows chaining `.pipe()` then `await`ing the [final result](#childprocessresult).
 
-#### kill(signalOrError?)
+#### kill(signal?, error?)
```

Comment:
The current signature does not make it clear you can pass `error` as the first argument.

```suggestion
#### kill(signal, error?)
#### kill(error?)
```

Reply:
That's better.